packages/browser/src/breadcrumbs/WebRequestEventSubscriber.ts

```diff
@@ -0,0 +1,70 @@
+import {
+    BreadcrumbLogLevel,
+    BreadcrumbsEventSubscriber,
+    BreadcrumbsManager,
+    BreadcrumbType,
+} from '@backtrace/sdk-core';
+
+export class WebRequestEventSubscriber implements BreadcrumbsEventSubscriber {
+    private _xmlHttpRequestOriginalOpenMethod?: typeof XMLHttpRequest.prototype.open;
+    private _fetchOriginalMethod?: typeof window.fetch;
+
+    public start(breadcrumbsManager: BreadcrumbsManager): void {
+        if ((breadcrumbsManager.breadcrumbsType & BreadcrumbType.Http) !== BreadcrumbType.Http) {
+            return;
+        }
+        const xmlHttpRequestOriginalOpenMethod = XMLHttpRequest.prototype.open;
+        XMLHttpRequest.prototype.open = function (
+            method: string,
+            url: string | URL,
+            async?: boolean,
+            username?: string | null,
+            password?: string | null,
```

Comment:
You could replace these with `...args: any[]` and pass them to `call`.

Reply:
Types complain a lot about missing the async parameter or the existence of it. Explained on slack